packages/ns-threat_shield/files/ts-ip-download

```diff
@@ -23,6 +23,7 @@ categories=$(uci -q get threat_shield.config.categories)
 if [ -z "$categories" ]; then
     exit 0
 fi
+categories="$categories whitelist"
```

Comment:
"whitelist" can be downloaded if the system has an active subscription
This line could be moved at 43

Reply:
The whitelist could be present also in other sources not only on nethesis ones.
If the category doesn't exists the download will fail but the process will continue.

In the end, I do not think we should restrict whitelist implementation only for Nethesis subscriptions.

Reply:
Can I move to the merge?

Reply:
Yes, go ahead :-)